openlibrary/coverstore/code.py

```diff
@@ -259,28 +269,22 @@ def notfound():
         if value is None or value in config.blocked_covers:
             return notfound()
 
-        if 8_820_000 > value >= 8_000_000 and size == "L":
-            # This item is currently offline due to heavy traffic;
-            # Fix incoming in the next ~week; See:
-            # - https://webarchive.jira.com/browse/PBOX-3879
-            # - https://github.com/internetarchive/openlibrary/issues/9560
-            size = "M"
-
         # redirect to archive.org cluster for large size and original images whenever possible
         if size in ("L", "") and self.is_cover_in_cluster(value):
             url = zipview_url_from_id(value, size)
             return web.found(url)
 
+        # This chunk is unique:
+        #   - they are stores as both .zip and .tar files (for now)
+        #   - the naming scheme is the older scheme used by .tar files
+        #     (eg item `l_covers_0008` instead of `olcovers9`
+        #   - the db is not yet updated with their correct URLs
+        if size == "L" and 8_820_000 > value >= 8_000_000:
```

Comment:
```suggestion
        if size in ("L", "") and 8_820_000 > value >= 8_000_000:
```
I am a bit unsure of this feedback as I confess I'm not entirely sure of the desired behavior. Should the `zip` be used for http://covers.openlibrary.org/b/id/8000001.jpg? If so, a change similar to this _may_ be necessary, though I may simply be misunderstanding the flow and the desired goal.

Reply:
Good Q! Only the L covers have had their tars converted to zips right now; this will expand to all sizes as more of the items are fixed up!